Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHLT_cff.py

```diff
@@ -15,11 +15,6 @@
 import RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi
 onlineBeamSpot = RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi.onlineBeamSpotProducer.clone()
```

Comment:
```suggestion
onlineBeamSpot = RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi.onlineBeamSpotProducer.clone(
    useBSOnlineRecords = True,
    timeThreshold = 999999
)
```

Otherwise this will fail at runtime.